pkg/util/helper/workstatus.go

```diff
@@ -61,14 +59,12 @@ func AggregateClusterResourceBindingWorkStatus(c client.Client, binding *workv1a
 	}
 
 	binding.Status.AggregatedStatus = aggregatedStatuses
-	err = c.Status().Update(context.TODO(), binding)
-	if err != nil {
-		klog.Errorf("Failed update clusterResourceBinding(%s). Error: %v.", binding.Name, err)
-		return err
-	}
-	klog.Infof("Update clusterResourceBinding(%s) with AggregatedStatus successfully.", binding.Name)
-
-	return nil
+	return retry.RetryOnConflict(retry.DefaultBackoff, func() (err error) {
+		if err = c.Get(context.TODO(), client.ObjectKey{Name: binding.Name}, binding); err != nil {
+			return err
+		}
```

Comment:
Why not move `binding.Status.AggregatedStatus = aggregatedStatuses` here? Just like L41.

Reply:
+1

Reply:
done